src/pages/createPost.tsx

```diff
@@ -154,122 +62,111 @@ const createPost = () => {
-              
+            <Stack spacing={2}>
+              <Autocomplete
+                id="free-solo-demo"
+                freeSolo
+                options={Location.map((option) => option.title)}
+                renderInput={(params) => <TextField {...params} placeholder="เลือกสถานที่" />}
+              />
+            </Stack>
             <div style={helperTextBox}>
               <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
               <FormHelperText sx={helperText}>0/100</FormHelperText>
             </div>
           </Grid>
-          
           <Grid item>
-
-          <Typography variant="h5">Date time</Typography>
-          <Grid container spacing={2}>
-          <Grid item xs={6}>
-              <Typography variant="body1">Start</Typography>
-              <LocalizationProvider dateAdapter={AdapterDayjs}>
-              <DateTimePicker
-          
-          value={value}
-          onChange={handleChange}
-          renderInput={(params) => <TextField {...params} fullWidth/>}
-          
-        /></LocalizationProvider>
+            <Typography variant="h5">Date time</Typography>
+            <Grid container spacing={2}>
+              <Grid item xs={6}>
+                <Typography variant="body1">Start</Typography>
+                <LocalizationProvider dateAdapter={AdapterDayjs}>
+                  <DateTimePicker
+                    value={value}
+                    onChange={handleChange}
+                    renderInput={(params) => <TextField {...params} fullWidth />}
+                  />
+                </LocalizationProvider>
                 <div style={helperTextBox}>
                   <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
                 </div>
-            
-            </Grid> 
-
-          <Grid item xs={6}>
+              </Grid>
+              <Grid item xs={6}>
                 <Typography variant="body1">End</Typography>
                 <LocalizationProvider dateAdapter={AdapterDayjs}>
-              <DateTimePicker
-          
-          value={value}
-          onChange={handleChange}
-          renderInput={(params) => <TextField {...params} fullWidth />}
-          
-        /></LocalizationProvider>
-                  <div style={helperTextBox}>
-                    <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
-                  </div>
-            
-          </Grid> 
-
-        </Grid>
-
+                  <DateTimePicker
+                    value={value}
+                    onChange={handleChange}
+                    renderInput={(params) => <TextField {...params} fullWidth />}
+                  />
+                </LocalizationProvider>
+                <div style={helperTextBox}>
+                  <FormHelperText sx={helperTextError}>ช่องนี้ไม่สามารถเว้นว่างได้</FormHelperText>
+                </div>
+              </Grid>
+            </Grid>
```

Comment:
do you handle "end date < start date" and "start date < current time" errors?
I think it is required.